Ch07/Ch07.md

```diff
@@ -392,34 +390,41 @@ can reach them.
 When you create a database, you also specify the number of replicas for that database. This is used by the cluster to decide how many database instances it should create
 for this database group. One of the roles of the supervisor is to make sure that this value is maintained over time. What do I mean by this?
 
-Well, consider a five-node cluster and a database that was created with a replication factor of three. The cluster assigned this database to nodes A, B and E. After a while,
-the supervisor notices that node E is down. It moves the node to the rehab program and hopes that it will recover. However, if it is down for too long (the default value is 15 minutes),
-the supervisor decides that it can't assume that node E will come up any time soon and it needs to take action to ensure we have enough replicas for the data
+Well, consider a three-node cluster and a database that was created with a replication factor of two. The cluster assigned this database to nodes A and B. After a while,
+the supervisor notices that node B is down. It moves the node to the rehab program and hopes that it will recover. However, if it is down for too long (the default value is 15 minutes),
+the supervisor decides that it can't assume that node B will come up any time soon and it needs to take action to ensure we have enough replicas for the data
```

Comment:
the supervisor will decide that it can't assume node B will come up any time soon and it will need to take action